rustls/src/msgs/handshake.rs

```diff
@@ -550,8 +550,11 @@ impl CertificateStatusRequest {
 
 // ---
 
+/// RFC8446: `PskKeyExchangeMode ke_modes<1..255>;`
 impl TlsListElement for PSKKeyExchangeMode {
-    const SIZE_LEN: ListLength = ListLength::U8;
+    const SIZE_LEN: ListLength = ListLength::NonZeroU8 {
+        empty_error: InvalidMessage::IllegalEmptyList("PskKeyExchangeModes"),
```

Comment:
I prefer the spelling you picked for the error string, but should we keep it in sync with the code?

Reply:
Added a renaming commit for this type.